### ISSUE TITLE
About Loss

### ISSUE THREAD

**Issue by HIHIHAHEI** [jiangyao-scu/JL-DCF-pytorch#3]
Hi, when did your loss begin to converge? This is my result, there are some problem, right?
![667d7c941f70c46fff643b90055c1e5](https://user-images.githubusercontent.com/40845081/120575348-7fc6d980-c453-11eb-8dd9-a6b61e451cc9.png)

**Comment by jiangyao-scu** [comment]
> Hi, when did your loss begin to converge? This is my result, there are some problem, right?
> ![667d7c941f70c46fff643b90055c1e5](https://user-images.githubusercontent.com/40845081/120575348-7fc6d980-c453-11eb-8dd9-a6b61e451cc9.png)

Thank you for paying attention to our work. The converged loss value after 45 epochs is around 7k and the loss value will fluctuate to a certain extent during training. You can monitor the loss curve by using Tensorboard to see whether the network is converging.
